### DISCUSSION TITLE
更新系统版本后不能自动切换深色和浅色模式

### DISCUSSION THREAD

**Discussion by visiLi** [AutoDarkMode/Windows-Auto-Night-Mode#449] [Q&A] [answered]
**系统版本如下：**
版本	Windows 11 专业版
版本	22H2
安装日期	‎2022/‎4/‎20
操作系统版本	22598.1
体验	Windows Feature Experience Pack 1000.22598.1.0

期待提供解决方案。

**Comment by visiLi** [comment, marked as answer]
临时解决方式：
[https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/discussions/435](url)

**Reply by Tree-New-Beee** [reply]
404?
